.azure-pipelines/build_steps.sh

```diff
@@ -25,6 +25,14 @@ conda install --yes --quiet conda-forge-ci-setup=2 conda-build -c conda-forge
 setup_conda_rc "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
 
 run_conda_forge_build_setup
+
+# Install the yum requirements defined canonically in the
+# "recipe/yum_requirements.txt" file. After updating that file,
+# run "conda smithy rerender" and this line will be updated
+# automatically.
+/usr/bin/sudo -n yum install -y mesa-libGL mesa-dri-drivers libselinux libXdamage libXxf86vm mesa-common-dev libglu1-mesa-dev
```

Comment:
> @ocefpaf, thanks for that resource. I've already found and done that, it didn't work.

@andyfaff you [forgot to re-render](https://conda-forge.org/docs/maintainer/knowledge_base.html#yum-requirements-txt).